redhat-cop/infra-ansible
# Adding option to serve ansible tower .zip file locally

### What does this PR do?
This change allows us to serve an ansible tower ,zip file locally. This can be useful in case where there are change(s) needed to be made to the .zip file before installing it (e.g.: local bug fixes, etc.). 

### How should this be tested?
Download the ansible tower .zip file (see URL in default/main.yml), then update the inventory to point to the local .zip file and set the new inventory var `ansible_tower_remote_src` to `false`. This will use the local file to install Ansible Tower. 

### Is there a relevant Issue open for this?
N/A

### Other Relevant info, PRs, etc.
Follow-up to PR #584 as an additional feature. 

### People to notify
cc: @redhat-cop/infra-ansible @jfilipcz
